docs/proposals/helm-charts.md

```diff
@@ -0,0 +1,169 @@
+and add the latest version of each chart to the `Charts` field as well as
+its name to the list of `imageNames` to be returned.
+
+After these image names are filtered, the subsequent call to `FetchSpecs()`
+will find the chart by name in the `Charts` field and create a `Spec` object
+for it. For example:
+
+```
+// Convert chart to Bundle Spec
+spec := &apb.Spec{
+    Runtime:     2,
+    Version:     "1.0",
+    Async:       "optional",
+    Bindable:    false,
+    Image:       r.Config.BaseImage,
+    FQName:      chart.Name,
+    Tags:        chart.Keywords,
+    Description: chart.Description,
+    Metadata: map[string]interface{}{
+        //"longDescription":  chart.Description,
+        "displayName":      fmt.Sprintf("%s (Helm)", chart.Name),
+        "documentationUrl": chart.Home,
+        "dependencies":     chart.Sources,
+        "imageUrl":         chart.Icon,
+    },
+    Plans: []apb.Plan{
+        apb.Plan{
+            Name:        "default",
+            Description: "Default plan for running helm charts",
+            Parameters: []apb.ParameterDescriptor{
+                apb.ParameterDescriptor{
+                    Name:      "repo",
+                    Title:     "Helm Chart Repository URL",
+                    Type:      "string",
+                    Default:   r.Config.URL.String(),
+                    Pattern:   fmt.Sprintf("^%s$", r.Config.URL.String()),
+                    Updatable: false,
+                    Required:  false,
+                },
+                apb.ParameterDescriptor{
+                    Name:      "chart",
+                    Title:     "Helm Chart",
+                    Type:      "string",
+                    Default:   chart.Name,
+                    Pattern:   fmt.Sprintf("^%s$", chart.Name),
+                    Updatable: false,
+                    Required:  false,
+                },
+                apb.ParameterDescriptor{
+                    Name:      "name",
+                    Title:     "Release Name",
+                    Type:      "string",
+                    Default:   "helmrunner",
+                    Updatable: false,
+                    Required:  false,
+                },
+                apb.ParameterDescriptor{
+                    Name:        "values",
+                    Title:       "Values",
+                    Type:        "string",
+                    DisplayType: "textarea",
+                    Default:     values,
+                    Updatable:   false,
+                    Required:    false,
+                },
+            },
+        },
+    },
+}
+```
+
+Most of the data required to create a useful `Spec` object is contained in
+the Chart object. However, in order to retrieve the default values we will use
+[Helm's Chartutil
+pkg](https://github.com/kubernetes/helm/blob/master/pkg/chartutil/load.go) to
+load the archive specified at `Chart.URLs[0]` and read the values.
```

Comment:
Can you elaborate on this process a bit more? What sort of archive is it? Are we downloading something and cracking it open? Where is it being stored? etc.